arduino/sensors/GYVL530X_LaserLightSensor/GYVL530X_LaserLightSensor.ino

```diff
@@ -0,0 +1,29 @@
+#include <Sensor2357.h>
+#include "Adafruit_VL53L0X.h"
+Adafruit_VL53L0X lox = Adafruit_VL53L0X();
+
+#define LOOP_DELAY_MS 10
+
+int readFloatValue(int min, int max) {
+  if (!lox.begin())
+  {
+    return -1;
+  }else{
+    VL53L0X_RangingMeasurementData_t measure;
+    lox.rangingTest(&measure, false);
+     if (measure.RangeStatus != 4){return measure.RangeMilliMeter;}
+     else {return 0;}  
+  }
+}
+
+SensorDevice_Seeed_XIAO_RP2040<1> device("LaserLightSensor");
+
+void setup() {
+  device.initSensor("intValue", readFloatValue, 0, 65535);
```

Comment:
This one too, instead of "intValue", how about "distanceMM" if it's in millimeters.

Reply:
Additionally, is the measurement in integer or floating point?